examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -0,0 +1,218 @@
+# Extract subjects variabilities around a robust group connectivity
+# -----------------------------------------------------------------
+
+###############################################################################
+# We can use **both** correlations and partial correlations to capture
+# reproducible connectivity patterns at the group-level and build a **robust**
+# **group connectivity matrix**. This is done by the **tangent** kind.
+tangent_measure = ConnectivityMeasure(kind='tangent')
+
+###############################################################################
+# We fit our ADHD group and get the group connectivity matrix stored as
+# in `tangent_measure.mean_`, and individual deviation matrices of each subject
+# from it.
+tangent_matrices = tangent_measure.fit_transform(adhd_subjects)
+
+###############################################################################
+# `tangent_matrices` model individual connectivities as
+# **perturbations** of the group connectivity matrix `tangent_measure.mean_`.
+# Keep in mind that these subjects-to-group variability matrices do not
+# straight reflect individual brain connections. For instance negative
+# coefficients can not be interpreted as anticorrelated regions.
+for n, matrix in enumerate(tangent_matrices[:4]):
+    plot_matrix(matrix, 'tangent variability, subject {0}'.format(n))
+
+plotting.plot_connectome(tangent_measure.mean_, msdl_coords,
+                         title='mean tangent')
+
+###############################################################################
+# Which connectivity kind is more powerful? A classification answer
+# -----------------------------------------------------------------
+
+###############################################################################
+# *ConnectivityMeasure* can output the estimated subjects coefficients
+# as a 1D arrays through the parameter *vectorize*.
+connectivity_biomarkers = {}
+kinds = ['correlation', 'partial correlation', 'tangent']
+for kind in kinds:
+    conn_measure = ConnectivityMeasure(kind=kind, vectorize=True)
+    connectivity_biomarkers[kind] = conn_measure.fit_transform(pooled_subjects)
+
+# For each kind, all individual coefficients are stacked in a unique 2D matrix.
+print('{0} correlation biomarkers for each subject.'.format(
+    connectivity_biomarkers['correlation'].shape[1]))
+
+###############################################################################
+# Note that we use the **pooled groups**. This is crucial for **tangent** kind,
+# to get the displacements from a **unique** `mean_` of all subjects.
+
+###############################################################################
+# We split the subjects in 3 folds, while considering between-sites variability
+from sklearn.cross_validation import StratifiedKFold
+
+classes = ['{0}{1}'.format(site_name, adhd_label)
+           for site_name, adhd_label in zip(site_names, adhd_labels)]
+cv = StratifiedKFold(classes, n_folds=3)
```

Comment:
I am giving both `adhd_labels` and `site_names` to constraint also by sites the splitting of data, no ? For instance I don't want ADHD patients to be exclusively from NYU site in one fold, and no NYU patients in others, because the matrices vary a lot between sites (actually they vary more across sites than between ADHD and controls ! that's why I think it is important to conserve the same sites proportion in each fold)
Do you mean I should rename `classes` to something like `subjects_labels` ?